### DISCUSSION TITLE
regolith-look-selector > gruvbox > gtk foreground color & dark mode

### DISCUSSION THREAD

**Discussion by simonfogliato** [regolith-linux/regolith-desktop#842] [General]
I love the gruvbox theme but what can I tweak to enforce dark mode similar to dracula? It seems in some cases the foreground and background are swapped.

dpkg --info gruvbox-gtk_1.0.1-1-1regolith_amd64.deb 
 new Debian package, version 2.0.
 size 178318 bytes: control archive=7239 bytes.
     293 bytes,    10 lines      control              
   40222 bytes,   431 lines      md5sums              
 Package: gruvbox-gtk
 Version: 1.0.1-1-1regolith
 Architecture: amd64
 Maintainer: Ken Gilmer <kgilmer@gmail.com>
 Installed-Size: 10742
 Section: x11
 Priority: optional
 Homepage: https://github.com/3ximus/gruvbox-gtk
 Description: Gruvbox Theme for GTK
  Created with oomox based on Gruvbox colors.

![Screenshot_20230413_081237](https://user-images.githubusercontent.com/16430413/231806659-f36df952-6985-4b5f-b5cb-6372d56c4928.png)
![Screenshot_20230413_081446](https://user-images.githubusercontent.com/16430413/231806672-fb899392-7532-46ef-9807-f0b7fc51c9ef.png)

**Comment by eduncan911** [comment]
You should show a screenshot of VSCodium...  Ick.


![Screenshot from 2024-01-21 17-38-54](https://github.com/regolith-linux/regolith-desktop/assets/104980/25908b5c-6301-4926-b50d-4f16a37d86ba)
